source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -50,14 +60,18 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::filter>,
   // Calls the callback with status.
   void doneWithStatus(const Status& status);
 
-  // Return true if it is OK to forward this request without JWT.
-  bool okToBypass() const;
+  // Start verification process. It will continue to eliminate bad tokens until it finds one to
```

Comment:
It's unclear what do you mean by 'bad'. Can you elaborate more

Reply:
updated the comment to replace bad tokens with tokens with invalid claims.